class/defaults.yml

```diff
@@ -151,6 +151,7 @@ parameters:
       vshn:
         enabled: false
         secretNamespace: ${crossplane:namespace}
+        mailgun_smtp_password: "?{vaultkv:${cluster:tenant}/${cluster:name}/appcat/mailgun_smpt_password}"
```

Comment:
We should probably document where this smtp password comes from.

Also, if we only have one password for mailgun, then maybe we should not store it in a cluster specific place? Then the manual work to install appcat will be a bit less.